.github/workflows/publish.yaml

```diff
@@ -84,11 +81,11 @@ jobs:
           TRACK=${{ needs.release_checks.outputs.track }}
           if [ ! -z "$RISK" ] && [ "${RISK}" != "no-risk" ]; then TAG=${TRACK}_${RISK}; else TAG=${TRACK}; fi
       
-          IMAGE_NAME="ghcr.io/canonical/charmed-karapace"
+          IMAGE_NAME="ghcr.io/deusebio/charmed-karapace"
```

Comment:
sorry, let over from testing [here](https://github.com/deusebio/charmed-karapace-rock/actions/runs/9595380772/job/26460006402). I'm going to update it